jobs/resource_job.go

```diff
@@ -74,6 +74,17 @@ type TaskDependency struct {
 	TaskKey string `json:"task_key,omitempty"`
 }
 
+// BEGIN Jobs + Repo integration preview
+type GitSource struct {
```

Comment:
simplify naming - we already inside `git_source`, why we need to repeat `git_` everywhere? we can just use `url`, `branch`, ...

Reply:
I am not sure I understood, do you think the field in HCL should be named 'url' instead of git_url?

```go
Url      string `json:"git_url" tf:"alias:url"`
```

Reply:
Very helpful, I agree. I imagine tf:"alias:url" sets the name of the property in HCL. Is there somewhere I can read about the struct tags supported by Terraform? I can't find googling it

Reply:
Yes, alias will be used as a name in HCL, while json will name the field in JSON object. Really, these things aren’t standard in terraform (although should be, imho) - they were added by Serge in Databricks provider only. Source code is in common/reflect_reaource

Reply:
Thanks, would you mind giving a look the structure? I am now writing unit tests for the PR